helm-charts/charts/kube-starrocks/charts/starrocks/templates/starrockscluster.yaml

```diff
@@ -370,3 +370,33 @@ spec:
       configMapName: {{template  "starrockscluster.cn.configmap.name" . }}
       resolveKey: cn.conf
 {{- end }}
+{{- if .Values.starrocksFeProxySpec.enabled }}
+  starRocksFeProxySpec:
+    {{- if .Values.starrocksFeProxySpec.image.repository }}
```

Comment:
what if use doesn't provide the repository setting, the crd will not have the image configuration, will operator fill the `image` field?

Reply:
in starrocksFeProxySpec, no fields is necessary.